pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -578,6 +587,18 @@ func (m *managerImpl) evictPod(pod *v1.Pod, gracePeriodOverride int64, evictMsg
 	m.recorder.AnnotatedEventf(pod, annotations, v1.EventTypeWarning, Reason, evictMsg)
 	// this is a blocking call and should only return when the pod and its containers are killed.
 	klog.V(3).InfoS("Evicting pod", "pod", klog.KObj(pod), "podUID", pod.UID, "message", evictMsg)
+	_, otelSpan := m.tracer.Start(ctx, "Eviction/EvictPod", trace.WithAttributes(
+		attribute.String("k8s.pod.uid", string(pod.UID)),
+		attribute.String("k8s.pod.name", pod.Name),
+		attribute.String("k8s.namespace.name", pod.Namespace),
+	))
+	otelSpan.AddEvent("Evicting pod", trace.WithAttributes(
+		attribute.String("k8s.pod.uid", string(pod.UID)),
+		attribute.String("k8s.pod.name", pod.Name),
+		attribute.String("k8s.namespace.name", pod.Namespace),
+		attribute.String("k8s.eviction.message", evictMsg),
```

Comment:
We should make those attributes consts and reuse them, I can follow-up on that or you as you wish.

Reply:
Oh I see we have them https://github.com/open-telemetry/opentelemetry-go/blob/b44a2bbc3661fef4eb20bf8d568b28fd71473ea3/semconv/v1.6.1/resource.go#L525-L538

Reply:
Good point! I change code to use semconv const.

Reply:
Done